packages/execution-environments/src/common/BaseSnapExecutor.ts

```diff
@@ -77,17 +78,25 @@ export class BaseSnapExecutor {
     );
   }
 
-  private errorHandler(error: Error, data = {}) {
+  private errorHandler(
+    reason: string,
+    originalError: unknown,
+    data: Record<string, unknown>,
+  ) {
+    const error = new Error(reason);
+
+    const _originalError: Error | undefined = constructError(originalError);
+
     const serializedError = serializeError(error, {
-      fallbackError,
-      shouldIncludeStack: true,
+      shouldIncludeStack: false,
```

Comment:
Is there a reason not to include the stack?

Reply:
The stack is not included because the `error` variable passed into `serializeError` will always be constructed from a string and not have a trace.